daemon/fetchers/base.py

```diff
@@ -0,0 +1,26 @@
+from abc import ABC, abstractmethod
+from typing import Dict, Any, NamedTuple
+from enum import Enum
+
+
+# TODO: figure out file layout, remove, import from accumulator
+class UsageUnit(Enum):
+    MILLISECONDS = "milliseconds"
+    BYTES = "bytes"
+    MILLISECONDS_SEC = "milliseconds_sec"
+
+
+class UsageKey(NamedTuple):
+    timestamp: int
+    resource_id: str
+    app_feature: str
+    unit: UsageUnit
```

Comment:
These are copy pasted from the usage accountant module. I think you should use that module instead of building the logic from the ground up.